src/components/org-users/controllers.test.tsx

```diff
@@ -1740,7 +1740,7 @@ describe('org-users test suite', () => {
       },
     );
 
-    expect(response.body).toContain('Updated a team member');
+    expect(response.body).toContain('eam member details successfully updated');
```

Comment:
I assume you meant to swallow the first character in all of the above :D

Reply:
let me fix this

Reply:
Sorry, managed to merge before you got around to it :(

Reply:
#728 done